tests/rest/admin/test_user.py

```diff
@@ -5503,3 +5542,99 @@ def test_redact_messages_all_rooms(self) -> None:
                     redaction_ids.add(event["redacts"])
 
         self.assertIncludes(redaction_ids, original_event_ids, exact=True)
+
+
+class GetInvitesFromUserTestCase(unittest.HomeserverTestCase):
+    servlets = [
+        synapse.rest.admin.register_servlets,
+        login.register_servlets,
+        admin.register_servlets,
+        room.register_servlets,
+    ]
+
+    def prepare(self, reactor: MemoryReactor, clock: Clock, hs: HomeServer) -> None:
+        self.admin = self.register_user("thomas", "pass", True)
+        self.admin_tok = self.login("thomas", "pass")
+
+        self.bad_user = self.register_user("teresa", "pass")
+        self.bad_user_tok = self.login("teresa", "pass")
+
+        self.to_kick = self.register_user("kick_me", "pass")
+        self.to_kick_tok = self.login("kick_me", "pass")
+
+        self.random_users = []
+        for i in range(4):
+            self.random_users.append(self.register_user(f"user{i}", f"pass{i}"))
+
+        self.room1 = self.helper.create_room_as(self.bad_user, tok=self.bad_user_tok)
+        self.room2 = self.helper.create_room_as(self.bad_user, tok=self.bad_user_tok)
+        self.room3 = self.helper.create_room_as(self.bad_user, tok=self.bad_user_tok)
+
+        self.helper.join(self.room1, self.to_kick, tok=self.to_kick_tok)
+
+    def test_get_user_invite_count_test_case(self) -> None:
+        # grab a current timestamp
+        invites_sent_ts = self.hs.get_clock().time_msec()
+
+        # bad user sends some invites
+        for room_id in [self.room1, self.room2]:
+            for user in self.random_users:
+                self.helper.invite(room_id, self.bad_user, user, tok=self.bad_user_tok)
+
+        # advance clock 48 hours
+        self.reactor.advance(ONE_DAY_MS * 2)
+
+        # get a timestamp beginning 24 hours ago
+        current_time = self.hs.get_clock().time_msec()
+        from_ts = current_time - (ONE_DAY_MS)
+
+        # fetch invites with timestamp, none should be returned
+        channel = self.make_request(
+            "GET",
+            f"/_synapse/admin/v1/users/{self.bad_user}/sent_invite_count?from_ts={from_ts}",
+            access_token=self.admin_tok,
+        )
+        self.assertEqual(channel.code, 200)
+        self.assertEqual(channel.json_body["invite_count"], 0)
+
+        # fetch using original timestamp, all should be returned
+        channel = self.make_request(
+            "GET",
+            f"/_synapse/admin/v1/users/{self.bad_user}/sent_invite_count?from_ts={invites_sent_ts}",
+            access_token=self.admin_tok,
+        )
+        self.assertEqual(channel.code, 200)
+        self.assertEqual(channel.json_body["invite_count"], 8)
+
+        # send some more invites, they should show up in addition to original 8 using original ts
+        for user in self.random_users:
+            self.helper.invite(self.room3, self.bad_user, user, tok=self.bad_user_tok)
```

Comment:
Nit: I find the `invite(...)` calls really hard to read without the keyword args. Up to you.

```suggestion
            self.helper.invite(self.room3, src=self.bad_user, targ=user, tok=self.bad_user_tok)
```